src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1035,6 +1035,23 @@ void createEntityWithDiskBaseModelAndWriteOut() {
             assertNotNull(rtest, "Entity does not exist");
         }
 
+        /**
+         * A cluster should be able to contain a link to one or more external KBs
+         */
+        @Test
+        void createClusterWithLinkAndConfidence() {
+            // create a valid entity and cluster
+            Resource typeAssertionJustification = makeTextJustification(model, "NYT_ENG_20181231",
+                    42, 143, system, 0.973);
+            Resource newEntity = makeEntity(model, getEntityUri(), system);
+            markJustification(addType(newEntity, SeedlingOntology.Person), typeAssertionJustification);
+            Resource cluster = makeClusterWithPrototype(model, getClusterUri(), newEntity, "handle", system);
+
+            // link cluster to a KB
+            linkToExternalKB(model, cluster, "freebase:FOO", system, .398);
+
+            testValid("Cluster with Link and confidence is valid.");
```

Comment:
To be in line with other `testValid` calls, how about, "Create a cluster with link and confidence."